pulp_smash/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -77,6 +79,39 @@ def _gen_distributor():
     }
 
 
+def _sync_and_publish_repo(server_config, importer=None, distributors=None):
```

Comment:
I originally copy/pasted its functionality from a different test and since you mentioned in the issue it might be nice to have a function that syncs a valid repo, I added it (and promptly neglected to use the function where I originally copied from). 

Personally, I think the two use cases diverged enough that having a function to share the task is not worth the trouble, so I'm more than happy to remove it.